.readthedocs.yml

```diff
@@ -1,6 +1,20 @@
-# See: https://docs.readthedocs.io/en/latest/yaml-config.html
+# See: https://docs.readthedocs.io/en/stable/config-file/v2.html#formats
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/source/conf.py
```

Comment:
I'm pretty sure that it automatically finds the conf.py, but I'm not opposed to keeping this here if you want to be explicit

Reply:
Yes, it's because if it is not installed explicitly it runs into a numpy problem generating the search indices for the HTML documentation.
```
dumping object inventory... /home/docs/checkouts/readthedocs.org/user_builds/pykeen/envs/16/lib/python3.7/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
```